src/features/pages/partners/deriv-prime/index.tsx

```diff
@@ -12,6 +14,9 @@ const DerivPrime = () => {
                 title="_t_Liquidity solutions provider | Deriv Prime_t_"
             />
             <PaymentAgentAffiliateNav />
+            <Flex.Box padding_block={'40x'} margin_block="20x">
```

Comment:
I think, you don't need this wrapper anymore..

Reply:
that's to give a view in this page @aswathy-deriv  will remove this once merged